src/lib.rs

```diff
@@ -58,18 +58,18 @@ extern crate fnv;
 extern crate mopa;
 #[cfg(not(target_os = "emscripten"))]
 extern crate pulse;
+extern crate rayon;
 #[cfg(not(target_os = "emscripten"))]
 extern crate rayon_core;
```

Comment:
Well `rayon` itself uses `rayon_core`, and reexports all parts of it, so using this parts from `rayon_core` is all the same.